pipeline-trigger/integration-test/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:lts-alpine as builder
+
+COPY ./src ./src
+COPY ./package*.json ./
+COPY ./.eslint* ./
+
+RUN npm ci
+RUN npm run lint-ci
```

Comment:
To make use of the Docker image layer cache you need to structure the command in another way.

```
COPY ./package*.json ./
RUN npm ci

COPY ./src ./src
COPY ./.eslint* ./
RUN npm run lint-ci
```